pkg/kubelet/kubelet.go

```diff
@@ -2075,3 +2089,9 @@ func (kl *Kubelet) GetCachedMachineInfo() (*cadvisorApi.MachineInfo, error) {
 	}
 	return kl.machineInfo, nil
 }
+
+// GetNodeVersionInfo returns version information (from Cadvisor) of current machine
+func (kl *Kubelet) GetNodeVersionInfo() (*cadvisorApi.VersionInfo, error) {
```

Comment:
I think we don't need this function, we can probably call cAdvisor directly above. We only export the information through status anyways.